packages/terra-notification-dialog/src/_ContentLayoutAsList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNamesBind from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+
+import styles from './ContentLayoutAsList.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * The list of strings to display in a list format.
+   */
+  items: PropTypes.arrayOf(PropTypes.string).isRequired,
+};
+
+const NotificationDialogContentList = ({ items }) => {
+  const theme = React.useContext(ThemeContext);
+
+  /* eslint-disable react/no-array-index-key */
+  return (
+    <ul className={cx('list', theme.className)}>
+      {items.map((item, index) => (<li className={cx('list-item')} key={index}>{item}</li>))}
+    </ul>
+  );
+  /* eslint-enable react/no-array-index-key */
+};
+
+NotificationDialogContentList.propTypes = propTypes;
+
+export default NotificationDialogContentList;
```

Comment:
I'm having a bit of a hard time understanding the use-case/advantages of exporting this to users. Can you provide a short description of the ideal use-case and scenario we'd recommend users use this component?

Reply:
We need this layout in the navigation-prompt when multiples are pending and they should be presented as a list. 

If we offload the layout styles to terra-application, we risk applying styles that do not match the version of terra-notification-dialog we are using and would need to pay close attending to padding and margin changes in notification-dialog and adjust terra-application accordingly. 

My thought is it was cleaner to add this layout & the associated styles here and use the layout in terra-application. Even though Neil had these styles related to notification-dialog (really terra-application's usage of it), I assumed these layout styles could/should be applied beyond the notification-prompt dialog use-case described above.